tests/plots/test_controller.py

```diff
@@ -26,6 +26,7 @@ def test_observation_plot_representation():
             "values": [2.85325093, 7.20311703, 21.38648991, 31.51455593, 53.56766604],
             "std": [0.1, 1.1, 4.1, 9.1, 16.1],
             "x_axis": [0, 2, 4, 6, 8],
+            "attributes": "Key1 Value1<br>Key2 Value2<br>",
```

Comment:
Was just wondering if this is part of some test or it will fail if `attributes` are no present.

Reply:
added assertion and squashed commits 👍